src/main/java/exceptions/DependencyParserException.java

```diff
@@ -1,6 +1,7 @@
 package exceptions;
 
 public class DependencyParserException extends RuntimeException{
+    private static final long serialVersionUID = 4L;
```

Comment:
tip for the future in intellij if you enable following inspection
https://prnt.sc/s4irzl
you can alt enter on a class to generate a serialversionUID like the following
https://prnt.sc/s4isoq